float32 MaxListeningRangeMeters     # avalue from 1.0 to 120.0. The default is 50.0
